appserver/node-express/lib/db-client/qnaDoc/patch.js

```diff
@@ -66,6 +116,25 @@ module.exports = function (txid, spec) {
     );
   };
 
+  /**
+   * Handle the post of a new answer to a Samplestack question. The posted
+   * answer is merged with an answer template object, a unique ID, contributor
+   * information, and the current time. The resulting object is written to the
+   * MarkLogic database. During the write, the answer count for the question is
+   * incremented and the last activity date is updated.
+   *
+   * @param  {String} txid The transaction ID.
+   * @param  {Object} contributor Contributor object. Example:
+   *   {"id":"cf99542d-f024-4478-a6dc-7e723a51b040",
+   *    "displayName":"JoeUser"}
+   * @param  {String} questionId The question ID.
+   * @param  {Object} spec Answer content with "comments" and "text" properties.
+   * The comments property is an empty array (TODO Why? Remove?). Example:
+   *   {"comments":[],
```

Comment:
Thanks. I see that the spec (and thus the comments) comes in (from the routing) from the req object, and the empty comments is not of db-client concern, so not necessary to include specifics here.